<?xml version="1.0"?>
<launch>
	<group ns="shooter">
		<rosparam file="$(find ros_control_boilerplate)/config/2024_shooter_and_pivot.yaml" command="load"/>
		<node name="shooter_server_2024" pkg="behaviors" type="2024_shooter_server.py" output="screen"/>
		<node name="shooter_pivot_server_2024" pkg="behaviors" type="2024_shooter_pivot_server.py" output="screen"/>
	</group>
</launch>